src/twisted/python/_textattributes.py

```diff
@@ -49,7 +53,14 @@ def __repr__(self):
 
 
     def __getitem__(self, item):
-        assert isinstance(item, (list, tuple, _Attribute, str))
+        if isinstance(item, unicode):
+            warnings.warn("Calling _Attribute.__getitem__ with a unicode/str"
+                          " object instead of a bytes object is deprecated"
+                          " since Twisted 16.5.0",
```

Comment:
.5.0 is released, should this string be updated?

Reply:
Changed this to use the automatic "NEXT" string per @twm's suggestion